docs/querying/basics.md

```diff
@@ -299,33 +326,34 @@ PromQL supports line comments that start with `#`. Example:
 
 ### Staleness
 
-When queries are run, timestamps at which to sample data are selected
+The timestamps at which to sample data, during a query, are selected
 independently of the actual present time series data. This is mainly to support
```

Comment:
I don't think this is correct.  I think an instant query will give you actual timestamps from the underlying series, unless you aggregate the data in which case I don't know what it will do.
A range query never gives timestamps from the underlying series; they are always a multiple of `step` away from `end`.
(and subqueries are a multiple of their `step` away from the epoch, although I don't understand why)

Reply:
Arrrrrggg, I linked the right thing but wrote `query selector` instead of `vector selector` . Updated suggestion. Perhaps it would help if these weren't so similar in the first place.